src/style.css

```diff
@@ -0,0 +1,65 @@
+html {
+  font-family: "Roboto", "Arial", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  width: 100%;
+  min-height: 60px;
```

Comment:
bad idia use height to header

Reply:
I think It's a good idea to use min-height for the header if we use flexbox to align items, min-height means that container will stretch if needed